install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -387,12 +387,12 @@ cni:
   # This can be useful if you want to manage your CNI
   # configuration outside of a Kubernetes environment. This parameter is
   # mutually exclusive with the 'cni.configMap' parameter.
-  # readCniConf: /host/etc/cni/net.d/05-cilium.conf
+  readCniConf: /host/etc/cni/net.d/05-cilium.conf
 
   # -- When defined, configMap will mount the provided value as ConfigMap and
   # interpret the cniConf variable as CNI configuration file and write it
   # when the agent starts up
-  # configMap: cni-configuration
+  configMap: cni-configuration
```

Comment:
This is not safe to set by default.

Reply:
I will not uncomment it for now since I am not sure what this helm option means and how to set it.